gsrs-core-entities/src/main/java/ix/ginas/exporters/SpreadsheetFormat.java

```diff
@@ -13,6 +14,10 @@ public SpreadsheetFormat(String extension, String displayname) {
 
     public abstract Spreadsheet createSpeadsheet(OutputStream out);
 
+    public abstract Spreadsheet createSpreadsheet(OutputStream out);
```

Comment:
This looks like a way to fix a typo, we should fix it better otherwise we have 2 conflicting method names that must be implemented.

Reply:
Yes; I wanted to fix the typo and also realized that people may be counting on createSpeadsheet so figured it was easy to create one overload that calls another.
Here are options I see:
1) undo my changes and keep the old method, including misspelling.
2) use the new spelling throughout. Notify all devs around NCATS+FDA and then in the wider community.
3) use the misspelled overload alongside the one that is spelled correctly.

Does that make sense?